### PR TITLE
Feature/writing sub

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description

Writing submission component to allow users to upload documents from their device. Added styling file. Added a reusable component, UploadDocs.js, to take care of the uploads. Added private route in index. Worked on Header.js to have the title to be dynamic through props. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/common/Header.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const Header = ({ displayMenu = true, ...props }) => {
           <Button icon={<MenuOutlined />} type="text" />
         </Dropdown>
       )}
-      <h1 className="header-text">STORY SQUAD</h1>
+      <h1 className="header-text">{props.title || 'STORY SQUAD'}</h1>
```
Comment (Schrese) [inline src/components/common/Header.js:48]: Good catch. I absolutely forgot about this part

**Review by JLC6290 — DISMISSED** [review]

File: src/index.js [hunk 2]
```diff
@@ -78,6 +82,12 @@ function App() {
             <StoryPrompt LoadingComponent={ChildLoadingComponent} />
           )}
         />
+        <SecureRoute
```
Comment (bramirez96) [inline src/index.js]: I forgot to take this out! Let's drop this before we deploy!

**Review by bramirez96 — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 3]
```diff
@@ -35,7 +36,9 @@ import { NotFoundPage } from './components/pages/NotFound';
 import { ParentDashboard } from './components/pages/ParentDashboard';
 import { ParentSettings } from './components/pages/FamilySettings';
 import { StoryPrompt } from './components/pages/StoryPrompt';
+import { WritingSub } from './components/pages/WritingSub';
 import LoginCallbackLoader from './components/common/LoginCallbackLoader';
+//import WritingSub from './components/pages/WritingSub/RenderWritingSub';
```
Comment (Schrese) [inline src/index.js]: Just getting rid of this comment

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Review by trevorjamesmartin — APPROVED** [review]

**Review by bramirez96 — APPROVED** [review]
Nice, thank you!

**Pull request merged**